gluoncv/data/kinetics400/classification.py

```diff
@@ -139,11 +139,15 @@ def __getitem__(self, index):
 
         directory, duration, target = self.clips[index]
         if self.video_loader:
+            if '.' in directory:
```

Comment:
@bryanyzhu When stored dataset in `~/.mxnet/datasets`, then there will no extension in video name, like `val/abseiling/0wR5jVB-WPk_000417_000427`, thus will cause error on opening file when reading video directly , am I right?

Reply:
Yes, you are right. This change is for reading videos directly. 

Another corner case is, not all videos have the same extension. For example, .mp4, .avi, .webm, .mkv, etc. So it is better to spell out the entire video name instead of just the prefix.